File: src/main/java/com/henry/xman/XMan.java
package com.henry.xman;

import com.henry.xman.ui.dev.DevPanel;
import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.actionSystem.PlatformDataKeys;
import com.intellij.openapi.project.Project;


public class XMan extends AnAction {

    @Override
    public void actionPerformed(AnActionEvent e) {
        Project project = e.getData(PlatformDataKeys.PROJECT);
        X.init(project);
        new DevPanel(project).show();
    }

}

File: src/main/java/com/henry/xman/XMan.kt
package com.henry.xman

import com.intellij.openapi.project.Project

/**
 * Date  2022/8/30.
 * Author henry.jia
 * Description
 */
object XMan {
    lateinit var project: Project

    @JvmStatic
    fun init(project: Project) {
        this.project = project
    }
}